contrib/parseq-batching/src/main/java/com/linkedin/parseq/batching/BatchingStrategy.java

```diff
@@ -1,16 +1,5 @@
 package com.linkedin.parseq.batching;
 
-import java.util.ArrayList;
```

Comment:
It would be great to avoid stylistic changes because it is hard to figure out what has really changed and what is just reformatting.

Reply:
Wait a second, in my local, testLongRunningBatchTaskSuccess should fail without my change, is it passing for you? Then something is wrong!

Reply:
Ignore my previous comment, that is right, testLongRunningBatchTaskSuccess works fine even with previous code since there is no task cancellation and ParTask will wait for the second task promise complete. testLongRunningBatchTaskFailure is the original bug where task cancellation makes ParTask promise complete quickly.